src/translations/strings.yaml

```diff
@@ -928,6 +928,19 @@ devices:
     value: My devices
     comment: Title for the menu bar when viewing the devices or device limit pages
 
+  locationTitle:
```

Comment:
@flodolo format is incorrect now, but will be resolved once #8864 is merged. These strings will fall under the [helpSheets label](https://github.com/mozilla-mobile/mozilla-vpn-client/pull/8864/files#diff-51d1d7d3681b31f83aa184349c06be4a6c4c5500b0219d98b4192a24778daf1fR931)